#pragma once

#include "public.h"

#include <yt/core/rpc/public.h>

namespace NYT {
namespace NGrpc {

////////////////////////////////////////////////////////////////////////////////

NRpc::IServerPtr CreateServer(TServerConfigPtr config);

////////////////////////////////////////////////////////////////////////////////

} // namespace NGrpc
} // namespace NYT
